bearer/bearer.go

```diff
@@ -204,7 +216,7 @@ func (b *Token) SetIat(iat uint64) {
 //
 // See also SetExp, SetNbf, SetIat.
 func (b Token) InvalidAt(epoch uint64) bool {
-	return !b.lifetimeSet || b.nbf > epoch || b.iat > epoch || b.exp < epoch
+	return b.Nbf() > epoch || b.Iat() > epoch || b.Exp() < epoch
 }
```

Comment:
I suggest renaming this method to `ValidAt` and inverse all conditions. I wouldn't say I like methods and variables with negative meanings in names. It is intricate to use them in code. I had experience with this method not so long time ago - it wasn't easy enough

We may mark the current as obsolete, then add a new method

Reply:
done